### PR TITLE
Export ethiopia data to DHIS2

### PR DESCRIPTION
**Pull request by govindkrjoshi**
Epic: https://app.shortcut.com/simpledotorg/epic/9756?group_by=none&vc_group_by=week&ct_workflow=all&cf_workflow=500000031

## Because

We want to export our reporting data in Ethiopia to their DHIS2 instance  

## This addresses

Adds an exporter, rake task and cron entry for the DHIS2 export

## Test instructions

The output of the export can be tested using the following rake command:  `rake dhis2:ethiopia_export`

### CODE REVIEW

File: config/schedule.rb [hunk 1]
```diff
@@ -112,6 +112,12 @@ def local(time)
   end
 end
 
+every 1.month, at: local("04:15 am"), roles: [:cron] do
+  if Flipper.enabled?(:ethiopia_dhis2_export)
```
Comment (olttwa) [inline config/schedule.rb:116]: Please add an entry to `feature-flags.md`.
IK this is a manual chore, maybe this file can be auto-populated by a bot that raises PR/merges to master every night/weekly.

**Review by olttwa — APPROVED** [review]
All looks good, just 1 documenting change.

File: spec/exporters/dhis2/ethiopia_exporter_spec.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+              period: period.to_s(:dhis2),
+              value: export_values[key]
+            }
+          end
+        end
+      end
+
+      data_value_sets = double
+      allow(Dhis2.client).to receive(:data_value_sets).and_return(data_value_sets)
+      expect(data_value_sets).to receive(:bulk_create).with(data_values: expected_values[0])
```
Comment (qptr) [inline spec/exporters/dhis2/ethiopia_exporter_spec.rb:54]: 👌

File: app/exporters/dhis2/helpers.rb [hunk 3]
```diff
@@ -0,0 +1,68 @@
+module Dhis2::Helpers
+  def self.current_month_period
```
[the remark this answers is not in the document]
Comment (govindkrjoshi) [reply app/exporters/dhis2/helpers.rb:2]: Will create a separate PR for this.

File: app/exporters/dhis2/helpers.rb [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    @current_month_period ||= Period.current.previous
+  end
+
+  def self.htn_controlled(region, period)
```
[the remark this answers is not in the document]
Comment (govindkrjoshi) [reply app/exporters/dhis2/helpers.rb:6]: Will rename in the PR refactoring BD exporters.

File: config/schedule.rb (hunk 1 above)
Comment (govindkrjoshi) [reply config/schedule.rb:116]: Nice catch, will do.

**Pull request merged**